src/main/java/seedu/address/logic/Logic.java

```diff
@@ -51,4 +52,9 @@ public interface Logic {
      * Set the user prefs' GUI settings.
      */
     void setGuiSettings(GuiSettings guiSettings);
+
+    /**
+     * Return the list of Statistical PieChart datapoints
```

Comment:
```suggestion
     * Return the list of Statistical PieChart datapoints.
```

Reply:
oh omg i did not realise i missed out all the fullstops thx for catching